### PR TITLE
Added Tags for Facebook and Twitter

### PR DESCRIPTION
**Pull request by Demilad**
Updated Open graph, added Twitter tags and cards

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,25 +1,44 @@
...
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="https://nigerialogos.netlify.com/">
+    <meta property="og:title" content="Nigeria Logos - An open source project">
+    <meta property="og:description" content="A free logo library for Nigerian companies">
+    <meta property="og:image" content="/images/Card.jpg">
+    <!-- Twitter -->
+    <meta property="twitter:card" content="summary_large_image">
+    <meta property="twitter:url" content="https://nigerialogos.netlify.com/">
```
Comment (onyekvchi) [inline index.html]: Any reason why this still links to Netlify? @Demilad

File: index.html [hunk 2]
```diff
@@ -1,25 +1,44 @@
...
+    <meta name="msapplication-TileImage" content="/images/favicon.ico/ms-icon-144x144.png">
     <meta name="theme-color" content="#ffffff">
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="https://nigerialogos.netlify.com/">
+    <meta property="og:title" content="Nigeria Logos - An open source project">
+    <meta property="og:description" content="A free logo library for Nigerian companies">
+    <meta property="og:image" content="/images/Card.jpg">
```
Comment (onyekvchi) [inline index.html]: Small thing, but why's this file name capitalized? @Demilad

Comment (Demilad) [reply index.html]: No reason in particular, does that cause any issues?

Comment (onyekvchi) [reply index.html]: nope, it's just inconsistent

File: index.html (hunk 1 above)
Comment (Demilad) [reply index.html]: Didn't know we had moved domain, changed now

**Pull request merged**